### PR TITLE
Provide unshaded pulsar-kafka-client

### PR DESCRIPTION
**Pull request by sijie**
Similar as the original pulsar-client. It is good to provide unshaded pulsar-kafka-client, so when migrating from kafka to pulsar, there would be a change that original kafka client will be colocated with the pulsar-kafka-client. Providing an unshaded pulsar-kafka-client will address this concern.

### CODE REVIEW

**Comment by merlimat** [comment]
@sijie We should probably update the docs in `./site/docs/latest/adaptors/KafkaWrapper.md` to mention this artifact as well.

**Comment by sijie** [comment]
@merlimat update the doc to include section how to use `pulsar-kafka-client-original`

File: site/docs/latest/adaptors/KafkaWrapper.md [hunk 1]
```diff
@@ -60,6 +60,21 @@ thing that needs to be adjusted is the configuration, to make sure to point the
...
+unshaded pulsar kafka client wrapper.
+
+```xml
+<dependency>
+  <groupId>org.apache.pulsar</groupId>
+  <artifactId>pulsar-client-kafka-original</artifactId>
+  <version>{{ site.current_version }}</version>
+</dependency>
+```
+
```
Comment (merlimat) [inline site/docs/latest/adaptors/KafkaWrapper.md]: If using this dependency, application would just need to change the `Producer` class from `org.apache.kafka.clients.producer.KafkaProducer` to `org.apache.kafka.clients.producer.PulsarKafkaProducer`

Comment (sijie) [reply site/docs/latest/adaptors/KafkaWrapper.md]: done updated the doc

**Pull request merged**